libsrc/TID1500Reader.cpp

```diff
@@ -145,8 +145,8 @@ Json::Value TID1500Reader::getMeasurements() {
               Json::Value laterality = Json::nullValue;
               if(node->getConceptName() == CODE_SCT_FindingSite)
                 laterality = getContentItem(CODE_SCT_Laterality, cursor);
-              else if(node->getConceptName() == CODE_SRT_FindingSite)
-                laterality = getContentItem(CODE_SRT_Laterality, cursor);
+              else if(node->getConceptName() == CODE_SCT_FindingSite)
```

Comment:
You should be careful replacing those codes in reader tools. In this specific example, the reader intentionally checks SRT and SCT codes, since it should be able to parse older documents that are using SRT codes. In this specific place, and in the next one, SRT codes should be preserved.

Reply:
You mean to have both SCT and SRT or just SRT version?

Reply:
If you take a look at the functionality of the code, it should make more sense. Can you look at that specific function and see what those condition checks are doing?

Reply:
I checked it out. CODE_SCT_* are macros defined in DCMTK sct.h file. The previous version used to read the CODE_SRT_* from srt.h header file. There is no build error for this new version. It seems like the code is populating a json file from a DSRDocument. As you already mentioned though, it is not going to read SRT codes anymore unless we keep both srt and sct.

Reply:
Right, there is no build error. Since the code is supposed to be able to read both the new SR documents that were created using the SCT codes for those concepts, and also the new SRT codes, reader has to check both.